BO4E/ENUM/WertBedeutung.cs

```diff
@@ -0,0 +1,32 @@
+using BO4E.meta;
+
+namespace BO4E.ENUM
+{
+    /// <summary>
+    /// Represents the Type of Wert (SG10 STS 9015)
+    /// </summary>
+    [NonOfficial(NonOfficialCategory.UNSPECIFIED)]
+    public enum WertBedeutung
+    {
+        /// <summary> 6 </summary>
+        VERTRAG,
+
+        /// <summary> 8 </summary>
+        MESSWERTQUALITÄT,
```

Comment:
Versuch mal in den Enum-Werten Umlaute zu vermeiden bitte

Reply:
dass das von joscha kommt hätte ich ja nie gedacht :D

Reply:
Oh. Habe ich uebersehen

Reply:
fix